clio-client/src/test/resources/testdata/gvcfMetadata

```diff
@@ -0,0 +1,10 @@
+{
+    "analysisDate" : "2017-08-09T14:21:57.995-04:00",
+    "contamination" : 0.75,
+    "gvcfMd5" : "gvcfMd5",
```

Comment:
This shouldn't be parseable by the client (we use snake case everywhere instead of camel case).

Reply:
Oops I missed the dispatch specs during the rebase so that's why it hasn't been yelling about parsing. Changing to snake case & adding the tests back now